app/src/main/res/menu/home_menu.xml

```diff
@@ -50,6 +50,10 @@
       android:title="@string/menu_jump"
       app:showAsAction="never" />
   <item
+      android:id="@+id/translations"
+      android:title="Translations"
```

Comment:
[this](https://github.com/quran/quran_android/blob/master/app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java#L1110) is where the custom adapter is being made. since `SpinnerAdapter` is a `BaseAdapter`, you can just override `getViewTypeCount` to return 2, and `getItemViewType` to return 0 except for the last row (the custom row you will add), in which case you'll return 1.

then, in `getView`, if `getItemViewType` is 1, show a different xml or just a `TextView` with "get more translations"